README.md

```diff
@@ -8,9 +8,13 @@
 </p>
 
 <p align="center">
+[📖 Documentation][docs]
```

Comment:
```suggestion
(📖 Documentation)[docs]
```

Reply:
https://github.com/aquasecurity/trivy/blob/main/README.md?plain=1#L11

Reply:
resolved by adding a newline between `<p>` and markdown link